### PR TITLE
Forward to multiple addresses

### LINKED ISSUE #24: Multiple forwarding addresses
e.g. separate with pipe `forward-email=food:niftylettuce@gmail.com|elusiveonion@gmail.com...`

### PR DESCRIPTION
**Pull request by bkeroack**
- Add support for multiple forwarding addresses.
- Adds a config option for max forwarding addresses for a single input address (defaults to 10).
- Adds a check that the forwarded address is not itself forwarded, to prevent cascading forwarding.

Fixes #24 

@niftylettuce I would appreciate some feedback on this. The tests don't pass yet, but I think that's due to my local not being configured properly to run them (no spamassasin, DKIM keys, etc).

### CODE REVIEW

**Comment by niftylettuce** [comment]
please do not add an eslint config file

we already use XO/prettier and XO already uses eslint

File: index.js [hunk 1]
```diff
@@ -252,10 +253,20 @@ class ForwardEmail {
...
-            const address = await this.getForwardingAddress(to.address);
+            let addresses = await this.getForwardingAddresses(to.address);
+            if (addresses.length > this.config.maxForwardedAddresses) {
+              if (this.config.maxForwardedAddresses > 0) {
+                addresses = addresses.slice(
+                  0,
+                  this.config.maxForwardedAddresses - 1
+                );
+              } else {
+                addresses = [];
```
Comment (hija) [inline index.js]: I think, this is a very special case - setting up a forwarding service without allowing any forwards. Maybe we should log this case, so it's easier for the service provider to understand why nothing gets forwarded? I think we should do the same for the case in which the maxForwardedAddresses are exceeded.

**Comment by rebelga** [comment]
I hope this gets implemented.  For me, a couple additional recipients are needed but without that I am forced to otherwise inferior alternatives.  Thnanks for this project and service.

**Comment by niftylettuce** [comment]
will do, just need time.  almost done with a major rewrite of this that improves performance and the service drastically.

**Comment by niftylettuce** [comment]
This is re-enabled again per my work with commit https://github.com/niftylettuce/forward-email/commit/3f84c922a008eac2cb3c8b094b9914d90044553d

Try it out and let me know if all works well!

**Pull request closed without merging**